code/src/App.js

```diff
@@ -1,9 +1,47 @@
 import React from 'react'
+import {Provider} from 'react-redux'
+import { combineReducers, createStore } from '@reduxjs/toolkit'
+
+import { todos } from 'reducers/todos'
+import { AddTodo } from 'components/AddTodo'
+import { TodoList } from 'components/TodoList'
+
+import {
+  Wrapper,
+  ProviderContainer
+} from '../src/components/Styling'
+
+const reducer = combineReducers({
+  todos: todos.reducer,
+})
+
+//Local storage
```

Comment:
Wow I noticed you managed to do this local storage, I didnt think about it at first. I will try adding it to my to do list aswell - not so nice in my project that as soon as you refresh the page all the tasks are gone haha. Well done! 💯